pkg/auth/agent_local_authz_handler.go

```diff
@@ -0,0 +1,84 @@
+package auth
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/restapi"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+/* AgentLocalAuthzHandler is the authorizer middleware that is being used for
+ * agent based installer (ABI) authentication cases
+ */
+type AgentLocalAuthzHandler struct {
+	cfg *Config
+	log logrus.FieldLogger
+	db  *gorm.DB
```

Comment:
Do you actually use any of these anywhere?

Reply:
Good point, not needed. Removed!